roboflow/config.py

```diff
@@ -71,13 +71,14 @@ def get_conditional_configuration_variable(key, default):
 TYPE_SEMANTIC_SEGMENTATION = "semantic-segmentation"
 
 DEFAULT_BATCH_NAME = "Pip Package Upload"
+DEFAULT_JOB_NAME = "Annotated via API"
 
 RF_WORKSPACES = get_conditional_configuration_variable("workspaces", default={})
 
 
 def load_roboflow_api_key(workspace_url=None):
     if os.getenv("RF_API_KEY") is not None:
-        return os["RF_API_KEY"]
+        return os.getenv("RF_API_KEY")
```

Comment:
Is this a convention we use anywhere else? I think we call this `ROBOFLOW_API_KEY`in most spots... though I think in some spots we look in `.env` also and in some we look for a `.roboflow_key` file in the directory tree.

Reply:
yeah I'll change it to `ROBOFLOW_API_KEY` much better. 
`RF_API_KEY` isn't even working anyway.